rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2108,14 +2113,7 @@ public Integer setLockWhitelistDisableBlockDelay(Transaction tx, BigInteger disa
     }
 
     public Coin getLockingCap() {
-        // Before returning the locking cap, check if it was already set
-        if (activations.isActive(ConsensusRule.RSKIP134) && this.provider.getLockingCap() == null) {
-            // Set the initial locking cap value
-            logger.debug("Setting initial locking cap value");
-            this.provider.setLockingCap(bridgeConstants.getInitialLockingCap());
-        }
-
-        return this.provider.getLockingCap();
+        return lockingCapSupport.getLockingCap().get();
```

Comment:
This might throw a `NoSuchElementException` if there is no value present. Maybe adding something like `.orElse` in order to ensure a value is always being returned. wdyt?

Reply:
This might be redundant since in the `lockingCapSupport.getLockingCap()` method we ensure this returns a valid value. If this method hasn't any value, it sets the locking cap value from `LockingCapConstants.getInitialValue()`. We can do it again in an `orElse(LockingCapConstants.getInitialValue())` method, but I think this is redundant. WDYT?

[Take a look at the code of this PR.](https://github.com/rsksmart/rskj/blob/0b5663dac1a2354405099e4d03ec7e1b1a70612e/rskj-core/src/main/java/co/rsk/peg/lockingcap/LockingCapSupportImpl.java#L39)

Reply:
I see, if it guranteed to always return a value there is no need to use an Optional here. wdyt?

Reply:
Make sense. I'll remove it.

Reply:
Done!

Reply:
@apancorb We got back the `Optional` since there's a case where we need to handle a `null` value. This is the workaround: `orElse(null)`, keeping the same behavior that existed previously.